workflow/metrics/leader_test.go

```diff
@@ -0,0 +1,44 @@
+package metrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func TestIsLeader(t *testing.T) {
+	_, te, err := createTestMetrics(
+		&Config{},
+		Callbacks{
+			LeaderState: func() bool {
+				return true
+			},
+		})
+	if assert.NoError(t, err) {
```

Comment:
we do this in a lot of places in the tests, but I feel like these conditionals should be inverted too.

won't block on that though since it's pre-existing style.

Reply:
or rather, these could all be replaced with `require.` statements instead of conditionals for the same effect but without nesting

Reply:
I prefer require